lib/apiservers/engine/backends/container.go

```diff
@@ -1543,6 +1544,64 @@ func ContainerSignal(containerID string, sig uint64) error {
 	return nil
 }
 
+func getClientIPAddrs() []netlink.Addr {
+	l, err := netlink.LinkByName(clientIfaceName)
+	if err != nil {
+		log.Errorf("Could not look up link from client interface name %s due to error %s",
+			clientIfaceName, err.Error())
+		return nil
```

Comment:
Yeah I suppose so. Gotta say, not a huge fan of Go's multiple return value pattern, as it breaks function composition :(